tmt/base.py

```diff
@@ -2130,6 +2130,22 @@ def lint_unknown_keys(self) -> LinterReturn:
 
         yield LinterOutcome.PASS, 'correct keys are used'
 
+    def lint_empty_env_files(self) -> LinterReturn:
+        """ P001: env files are not empty """
+
+        env_files = self.node.get("environment-file") or []
+        if len(env_files) > 0:
```

Comment:
`if env_files:` is the Python way of checking list emptiness.

Reply:
Updated, thanks !